internal/proxy/github.go

```diff
@@ -175,7 +177,11 @@ func (p *Proxy) ProcessBuffer() {
 								break
 							}
 
-							if resp.StatusCode != 200 && resp.StatusCode != 404 { //Retry later if the response is not 200
+							if resp.StatusCode == 200 {
+								log.WithField("ns", ns).Infof(fmt.Sprintf("Request for %s to %s forwarded.", ns, req.Host))
+							} else if resp.StatusCode == 404 || resp.StatusCode == 400 { //400 - missing payload
+								log.Warn(fmt.Sprintf("Got status %s after retrying request on %s, throwing away the request", resp.Status, req.URL))
```

Comment:
same, please use Warnf instead of Warn(fmt.Sprintf( .. ))

Reply:
if the above is a return, then this can be changed to `if` instead of `else if`. Also, can't we return?

Reply:
ProcessBuffer keeps checking requests and keeps processing it. It never returns.